app/service/rpc_service.ts

```diff
@@ -163,6 +163,26 @@ class RpcService {
     }
     init.headers = { "Content-Type": "application/proto" };
     try {
+      if (capabilities.config.streamingHttpEnabled) {
+        init.headers["Content-Type"] = "application/grpc+proto";
+        init.body = lengthPrefixMessage(requestData);
+
+        const reader = (await this.fetch(url, "stream", init))?.getReader();
+        while (true) {
+          let result = await reader?.read();
+          if (result?.done) {
+            break;
```

Comment:
should this be `return` ? Seems like we'll do another `fetch` after this `break`, guessing that's not intended?

Reply:
Good catch, I am struggling with `return`s today lol.

I tested this a bunch without the fallback case at the bottom - and then added the fallback to support hiding this behind a flag, and guess I didn't notice the extra fetch.

Reply:
Fixed